### ISSUE TITLE
should fail if the element is not focusable

### ISSUE THREAD

**Issue by luciferankon** [getgauge/taiko#620]
### Expected behavior
should throw an error if the element is not focusable

### Actual behavior
It emits a success event
```✔ Focussed on the button with label Click Me!```

### Steps to reproduce
1. copy the following HTML in a file 
```
<!DOCTYPE html>
<html>
<body>
<input type="text" name="something" >something</input>
<button type="button" disabled>Click Me!</button>
</body>
</html>
```
2. add the following file in the script
```
const { openBrowser, goto, textBox, into, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto(filePath to html file);
        await focus(textBox("something"))
        await focus(button('Click Me!'))
        await write('something')
    } catch (error) {
        console.error(error);
    } finally {
      await closeBrowser();
    }
})();
```
3. run it using the 
` taiko script.js`

### Versions
```
node: v12.3.1
taiko: Version: 0.8.0 (Chromium: 76.0.3803.0) c4bd3810aa5790692479555507e40dc66a2a4fe3

**Comment by nehashri** [comment]
Fix should be available with e40d68045eb13d10ba6d64d32e835b9479e8f3c7

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the commit version ```e40d680```
